src/arch/arm/config.cmake

```diff
@@ -179,6 +179,17 @@ config_option(
     DEFAULT_DISABLED OFF
 )
 
+config_option(
+    KernelAArch64PassSError AARCH64_PASS_SERROR 
```

Comment:
There is a trailing white space here.

Reply:
I tried fixing the other complaints as well, amended the commit and pushed it. (In hindsight I should have created a branch for this pull request.) This means the help text will be messed up like all the others, but that should be fixed for all help texts instead of just one.

Is there a way to do these style checks locally before commit instead of finding out about it afterwards?

Reply:
You could run the script from https://github.com/seL4/seL4_tools/blob/master/misc/style.sh

Reply:
Yes, that's the one. I tend to use https://github.com/seL4/seL4_tools/blob/master/misc/style-all.sh to get all files so I have to think less about which files I changed, but the effect is the same.

Reply:
Thanks, calling seL4_tools/misc/style-all.sh from my seL4's root directory works. I also copied .gitlint and ran "gitlint install-hook" to get pre-commit message checking.